.coderabbit.yaml

```
@@ -0,0 +1,90 @@
+    - "coverage/**"
+    - ".tmp/**"
+    - "*.log"
+    - "package-lock.json"
+    - ".git/**"
+    - ".vscode/**"
+    - ".idea/**"
+    - "*.min.js"
+    - "*.min.css"
+    - "public/vendor/**"
+    - "static/vendor/**"
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)